signac/contrib/project.py

```diff
@@ -78,13 +78,13 @@ class Project:
     A :class:`Project` may only be constructed in a directory that is already a
     signac project, i.e. a directory in which :func:`~signac.init_project` has
     already been run. If project discovery (searching upwards in the folder
-    hierarchy until a project root is discovered) is desired, users should
+    hierarchy until a project is discovered) is desired, users should
```

Comment:
Can we say how a project is discovered? Like the presence of a `signac.rc` file?

Reply:
I would need to consult the implementation to refresh myself but I think it's defined by the existence of a `.signac/config` file (?). Either way, it may be better to write this definition in one place (not in every method/function docstring), and it's outside the scope of this PR which is focused on an API change.

Reply:
Agreed https://github.com/orgs/glotzerlab/projects/5#card-81869156